packages/patternfly-4/react-charts/src/components/ChartDonut/examples/ChartDonut.md

```diff
@@ -17,12 +19,12 @@ Learn to build a donut chart using a Katacoda tutorial starting with a simple ch
 [Start course](https://katacoda.com/patternfly/courses/charts/donut-chart)
 
 ## Examples
-```js title=Simple-donut-chart
+```js title=Basic
```

Comment:
I don't care if we use Simple or Basic in the title, but just want to be consistent. The title was "Simple donut chart"

Reply:
To be consistent with what we've done for the rest of Core/React documentation, example titles should not include the name of the component. There is an `<h1>` at the top of the page with the component name, and the site user knows which component they're looking at.

The rule of thumb I've been following is to make a title that makes sense in the context of `${ComponentName} ${ExampleTitle} Example`. For instance, "Donut chart basic example."

Once we get a technical writer onboard they can overrule this or do whatever they please, but I think it's good to stay consistent for now.